Classes/Settings/SettingsViewController.swift

```diff
@@ -154,19 +154,25 @@ NewIssueTableViewControllerDelegate, DefaultReactionDelegate {
     }
 
     func onReportBug() {
-        guard let viewController = NewIssueTableViewController.create(
-                client: GithubClient(userSession: sessionManager.focusedUserSession),
-                owner: "GitHawkApp",
-                repo: "GitHawk",
-                signature: .bugReport
-            ) else {
-                Squawk.showGenericError()
-                return
+        showTemplateOptions()
+    }
+
+    func showTemplateOptions() {
```

Comment:
In my understanding, you are not using this `func` anywhere else except `onReportBug`. Why not putting the implementation in `onReportBug`?